docs/installation/other_inst_scenarios/install_epinio_on_rancher.md

```diff
@@ -1,40 +1,51 @@
 ---
-sidebar_label: "Installing Epinio On Rancher"
+sidebar_label: "Installing Epinio on Rancher"
 sidebar_position: 17
-title: ""
+title: "Installing Epinio on Rancher"
+description: How to install Epinio on Rancher
+keywords: [epinio, kubernetes, rancher]
 ---
 
-# Installing Epinio On Rancher
+Before installing Epinio you need a running Rancher instance.
+If necessary use the Rancher [installation guides](https://rancher.com/docs) to set one up.
 
-Before installing Epinio, you need a running Rancher instance.
+<!--TODO:Confirm versions?-->
+This How-to was written using:
 
-If you do not have one, please use the Rancher's [installation guides](https://rancher.com/docs) to set one up.
-
-This How-to was written using the following versions:
-* [epinio helm chart 1.7.1](https://github.com/epinio/helm-charts/releases/tag/epinio-1.7.1)
-* Rancher 2.7.1
+- [epinio helm chart 1.7.1](https://github.com/epinio/helm-charts/releases/tag/epinio-1.7.1)
+- Rancher 2.7.1
 
 ## Kubernetes cluster
 
-The two supported methods are to either deploy a new cluster or to import an existing one into Rancher. The steps to take for either method are in the Rancher documentation (See `Cluster Provisioning`).<br />
-*While it is possible (for development) to deploy Epinio on the same node where Rancher is installed, this requires more steps and is not supported by Rancher.*
+The two supported methods are: to deploy a new cluster or to import an existing one into Rancher.
+<!--TODO:A direct link would be better if possible. But I can't seee an obvious one.-->
+The steps for either method are in the Rancher documentation (See `Cluster Provisioning`).
 
+:::note
+For development it's possible to deploy Epinio on the same node as Rancher.
+However, this needs more steps and isn't supported by Rancher.
+:::
 
 ## Epinio Prerequisites
 
 ### Install an Ingress controller
 
-If there's no Ingress controller running yet on the cluster, you need to install one. In the example below, Traefik is installed.
+If there's no Ingress controller running yet on the cluster, you need to install one. For the example below, we use Traefik.
+
+:::note
 
-> NOTE: If you are using the rancher dashboard in the browser, it is likely you already have an Ingress controller running.
+If using the Rancher dashboard in a browser, it's likely you already have an Ingress controller running.
 
-Go to `Apps` > `Repositories` > `Create`.
+:::
+
+<!--TODO: This is for when in the Rancher dashboard, right?-->
```

Comment:
Based on the following text I believe that this is true.

Reply:
Thanks for confirming.